packages/design-system/src/components/primitives/dnd/use-move.ts

```diff
@@ -1,20 +1,5 @@
-/* eslint-disable prefer-const */
-/* eslint-disable func-style */
-/*
- * Copyright 2020 Adobe. All rights reserved.
- * This file is licensed to you under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License. You may obtain a copy
- * of the License at http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
we can't remove the copyright like that! this is a serious violation of the license

Reply:
I will ask around, we may need to also add a second license file to the root

Reply:
we also need to: `You must cause any modified files to carry prominent notices
          stating that You changed the files;`
          
https://github.com/adobe/react-spectrum/blob/main/LICENSE#L97

yeah and we need to put that LICENSE file in the root too

Reply:
I was linked on twitter to how properly attribute  https://infra.apache.org/licensing-howto.html

Reply:
Makes me want to refactor use-drag to use browser events directly 😄

Reply:
yeah its not too far off, there isn't much logic